docs/src/snap/howto/refresh-certs.md

```diff
@@ -0,0 +1,98 @@
+# Refreshing Kubernetes Certificates
+
+Keeping your {{product}} cluster secure and functional requires
+regularly refreshing its certificates. Certificates in Kubernetes ensure
+secure communication between the various components of the cluster. If
+these certificates expire, it can lead to communication failures, disrupted
+services, and potential security risks. This how-to will walk you through
+the steps to refresh the certificates for both control plane and worker
+nodes in your {{product}} cluster.
+
+## What you will need
+- A running {{product}} cluster
+
+```{note} To refresh the certificates in your cluster, make sure it was
+initially set up with self-signed certificates or with your own CA certificates
+and keys during the bootstrap process.
+```
+
+### Refreshing Control Plane Node Certificates
+To refresh the certificates on control plane nodes, follow these steps:
+
+1. Access each control plane node in your cluster.
+2. Run the `refresh-certs` command:
+
+```bash
+sudo k8s refresh-certs --expires-in 1y --extra-sans mynode.local
+```
+
+This command refreshes the certificates for the control plane node, adding an
+extra Subject Alternative Name (SAN) to the certificate. The `--expires-in`
+flag sets the certificate's validity duration, which can be specified in years,
+months, days, or any unit accepted by the [ParseDuration][] function in Go.
+
+```{note} Ensure that you provide the same SANs that were used when the cluster
+was bootstrapped. If you don't, the control plane may fail to communicate with
+other nodes in the cluster.
```

Comment:
I'm curious. Why does the user need to provide the extra-sans if they are the same as the bootstrap sans?

Reply:
We are not storing the bootstrap SANs, as these are node-specific. The user might want to change the extra SANs when refreshing the certificates, and for orchestration layers (CAPI or Juju), they already know about the cluster-wide extra SANs we want to add.

Reply:
In the comment you write `The user might want to change the extra SANs when refreshing the certificates` but the text says: `Ensure that you provide the same SANs that were used when the cluster was bootstrapped.` Maybe at an "Ensure that you provide at least..."

Reply:
@louiseschmidtgen It's possible to check the current SANs. Let's add a snippet to check this!
@bschimke95 Your are absolutely right, let me rephrase that to convey that the user should include the host specific SANs.